scripts/deploy.py

```diff
@@ -261,9 +262,32 @@ def _execute_deployment():
         if build_process.returncode > 0:
             raise Exception('Build failed.')
 
+        # Update indexes, then prompt for a check that they are all serving
+        # before continuing with the deployment.
+        assert os.path.isfile(INDEX_YAML_PATH)
+        subprocess.check_output([
+            common.GCLOUD_PATH, '--quiet', 'datastore', 'indexes', 'create',
+            INDEX_YAML_PATH, '--project=%s' % APP_NAME])
+        datastore_indexes_url = (
+            'https://console.cloud.google.com/datastore/indexes?project=%s' %
+            APP_NAME)
+        common.open_new_tab_in_browser_if_possible(datastore_indexes_url)
+        while True:
```

Comment:
Why do we need to loop here? I see that after the input stage (which is a blocking statement), there are 2 outcomes both of which terminate the loop... So why loop in the first place? (Am I missing something here?)

Reply:
This is to catch the case when the user just presses Enter without typing anything.

Reply:
Ah that's good. Didn't notice the elif....